docs/griptape-framework/drivers/event-listener-drivers.md

```diff
@@ -14,26 +14,27 @@ import os
 
 from griptape.drivers import AmazonSqsEventListenerDriver
 from griptape.events import (
-    EventListener,
+    EventListener, event_bus
 )
 from griptape.rules import Rule
 from griptape.structures import Agent
 
-agent = Agent(
-    rules=[
-        Rule(
-            value="You will be provided with a block of text, and your task is to extract a list of keywords from it."
-        )
-    ],
-    event_listeners=[
+event_bus.add_event_listeners(
+    [
         EventListener(
```

Comment:
Does this need to be plural? Can there be a singular `add_event_listener` too?

Reply:
It does not _need_to be plural, no. Let's just leave it as is, docs will get an overhaul at some point soon.